images/hubble-proto/Makefile

```diff
@@ -0,0 +1,12 @@
+# Copyright 2017-2020 Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+include Makefile.defs
+
+.PHONY: image
+image:
+	docker build -t cilium/hubble-proto:${version} .
```

Comment:
nit: `Makefile.defs` defines `CONTAINER_ENGINE?=docker` to allow users to overwrite the docker CLI path. All other Makefile use `$(CONTAINER_ENGINE) build`, can we use this here as well?

Reply:
One issue with `Makefile.defs` is that it carries a lot of non-portable shell invocations that are broken on macOS.

Reply:
Ah, my bad, I confused the imported `Makefile.defs` with the global one. In that case I'm fine to leave it as is.